src/test/regress/expected/multi_join_order_additional.out

```diff
@@ -154,10 +156,12 @@ EXPLAIN SELECT count(*) FROM lineitem, orders
 	WHERE (l_orderkey = o_orderkey AND l_quantity > 5)
 	OR (l_orderkey = o_orderkey AND l_quantity < 10);
 LOG:  join order: [ "lineitem" ][ local partition join "orders" ]
-                         QUERY PLAN                         
-------------------------------------------------------------
- explain statements for distributed queries are not enabled
-(1 row)
+                                QUERY PLAN                                
+--------------------------------------------------------------------------
+ Aggregate  (cost=0.00..0.00 rows=0 width=0)
+   ->  Custom Scan (Citus Task-Tracker)  (cost=0.00..0.00 rows=0 width=0)
+         explain statements for distributed queries are not enabled
```

Comment:
I think "statement" is also used.
"There is no EXPLAIN statement defined in the SQL standard."
From: https://www.postgresql.org/docs/9.6/static/sql-explain.html